.env.example

```
@@ -17,6 +17,8 @@ NEXT_PUBLIC_APP_NAME=DL Starter
 
 # Feature Flags
 NEXT_PUBLIC_FLAG_BETA_FEATURE=false
+# Analytics example feature (disabled by default - see examples/analytics/)
+NEXT_PUBLIC_ENABLE_ANALYTICS=false
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_ENABLE_ANALYTICS key should go before the NEXT_PUBLIC_FLAG_BETA_FEATURE key

(UnorderedKey)